nodeup/pkg/model/kubelet_test.go

```diff
@@ -397,11 +397,13 @@ func RunGoldenTest(t *testing.T, basedir string, key string, builder func(*Nodeu
 }
 
 func Test_BuildComponentConfigFile(t *testing.T) {
+	maxPulls := int32(5)
 	componentConfig := kops.KubeletConfigSpec{
 		ShutdownGracePeriod:             &metav1.Duration{Duration: 30 * time.Second},
 		ShutdownGracePeriodCriticalPods: &metav1.Duration{Duration: 10 * time.Second},
 		ImageMaximumGCAge:               &metav1.Duration{Duration: 30 * time.Hour},
 		ImageMinimumGCAge:               &metav1.Duration{Duration: 30 * time.Minute},
+		MaxParallelImagePulls:           &maxPulls,
```

Comment:
This should make it simpler to read.
```suggestion
		MaxParallelImagePulls:           fi.PtrTo(int32(5)),
```